cmd/template/cluster/provider/templates/azure/azure_cluster.yaml.tmpl

```diff
@@ -0,0 +1,17 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: AzureCluster
+metadata:
+  labels:
+    "release.giantswarm.io/version": "{{ .Version }}"
+    "giantswarm.io/cluster": "{{ .Name }}"
+    "cluster.x-k8s.io/cluster-name": "{{ .Name }}"
+    "giantswarm.io/organization": "{{ .Owner }}"
+    "cluster.x-k8s.io/watch-filter": "capi"
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  location: ""
```

Comment:
```suggestion
```

Reply:
Can't remove it. `apply` fails with validation error.

Reply:
```
error: error validating "STDIN": error validating data: ValidationError(AzureCluster.spec): missing required field "location" in io.x-k8s.cluster.infrastructure.v1alpha4.AzureCluster.spec; if you choose to ignore these errors, turn validation off with --validate=false
```
Despite the defaulting